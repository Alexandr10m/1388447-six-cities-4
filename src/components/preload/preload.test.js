import React from "react";
import renderer from "react-test-renderer";
import Preload from "./preload.jsx";


describe(`Snapshot of Preload`, () => {
  it(`Should render correct`, () => {
    const tree = renderer
      .create(<Preload/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
