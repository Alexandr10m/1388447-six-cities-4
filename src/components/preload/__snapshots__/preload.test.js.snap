// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot of Preload Should render correct 1`] = `
<div>
  ... in progress
</div>
`;
